plugin/trino-mariadb/src/main/java/io/trino/plugin/mariadb/MariaDbClient.java

```diff
@@ -457,12 +457,12 @@ public void renameColumn(ConnectorSession session, JdbcTableHandle handle, JdbcC
                     quoted(newRemoteColumnName));
             execute(connection, sql);
         }
-        catch (TrinoException e) {
+        catch (SQLSyntaxErrorException syntaxError) {
```

Comment:
was confusion when I first read this - the change is equivalent because now the original exception gets thrown by `execute` instead of wrapped within TrinoException.